<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Orbit_calc</class>
 <widget class="QMainWindow" name="Orbit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbit_calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Button_add">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button_clear">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Button_equal">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color : gray;
	border : 1px solid gray;
	border-radius : 5%;
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Button_div">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Button_mem_clear">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Button_sub">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Button_mult">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Button_mem_rec">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="Button_mem_set">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #800000;
	border : 1px solid #b22222;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff;
}

QPushButton:pressed {
	background-color : #b22222;
	border : 1px solid #8b0000;
	border-radius : 5%;
	padding : 7px;
	color : #ffffff
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button_point">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color : #c0c0c0;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}

QPushButton:pressed {
	background-color : #a9a9a9;
	border : 1px solid gray;
	border-radius : 5%;
	padding : 7px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
